doc/Manual.md

```diff
@@ -0,0 +1,637 @@
+
+While contest loggers on other platforms have adopted the standard windowed interface modern users expect, Linux contest loggers have all gravitated to console-based designs that operate in a terminal window. There are a couple of reasons for that. 
+
+First, Linux users tend to be relatively comfortable using the terminal. Second, if we take a step back and look at the design of a contest logger objectively, a windowed interface is actually a handicap. Ideally, a contester's hands never have to leave the keyboard - and what better way to ensure that than to put the whole logger in a console?
+
+Console-based loggers are also fast, computationally lightweight, and easy to configure for remote operation with limited bandwidth. TLF can run on anything from a high-end desktop with triple monitors to a headless Raspberry Pi Zero.
+
+The chief disadvantage of this design is that there's a steep learning curve for new users. TLF is a powerful and efficient tool for contest logging, but you can't just click through the menus and figure things out on your own. You need to read some documentation. This manual, plus the built-in man page (accessed in the terminal with `man tlf`) is that documentation. Once you've gotten familiar with TLF's operation, you may begin to wonder why anyone would design a contest logger any other way.
+
+## About the name
+
+Why "TLF?" It's an old joke about someone sending bad CW (Morse code) over the radio, as if stomping on the key with their foot. TLF is "try left foot instead."
+
+## Getting started
+
+This manual assumes you've read the main `README.md` file and gotten through basic installation of TLF. Two important concepts from that document are:
+
+* TLF configuration happens entirely in the `logcfg.dat` and rules files, and local copies of those files will override the defaults.
+* Creating a new directory for each contest, with its own `logcfg.dat` and rules files, will help keep things organized.
+
+Note that the rules file has to be in a subdirectory called `rules` in order for TLF to recognize it. The rules file itself will be named for the contest, with no extension, e.g. `arrldx_usa` is the rules file for a USA-based station operating in the ARRL DX contest (either CW or SSB). In the image below, the arrltest.log and tlfmarkers files were created by TLF on its initial launch from this directory. The former is the working log file, and the latter is for plotting DX spots on a grayline map displayed in a separate window (optional). 
+
+<img title="Screenshot" alt="Screenshot showing basic file structure." src="images/BasicFileStructure.png" />
```

Comment:
Is there a way to view images on a text console? All other parts of the MD file are perfectly viewable using a plain text editor.

Reply:
I fear that will not be possible in general.  The above file structure may be shown by ascii art.  For other images that may be not so easy.

In relation to that we should add some rules to the makefile to generate a pdf formatted manual from the sources. That way after installation you do not have to rely to using a web browser for reading the manual.
I tried pandoc with the weasyprint backend.  The results looked good. Not sure if there is an easier way.